desc/objectives/_generic.py

```diff
@@ -513,8 +518,8 @@ def __init__(
         target=None,
         bounds=None,
         weight=1,
-        normalize=True,
-        normalize_target=True,
+        normalize=False,
```

Comment:
why this change?

Reply:
I had changed the default to `normalize=False` for all the objectives that are already dimensionless, but I realize that is unnecessary. I will revert the change.